packages/core/src/vuePlugins/actions/useExtendedMasterportalapiMarkers.ts

```diff
@@ -0,0 +1,117 @@
+import { Map, Feature, MapBrowserEvent } from 'ol'
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+const fallbackHoverFill = '#008A8A'
+const fallbackSelectionFill = '#00A95C'
+
+export const imgSize = [23, 32]
+export const imgSizeMulti = [35, 32]
+
+const prefix = 'data:image/svg+xml,'
+
+/* NOTE path below copied and adapted from
+ * @masterportal/masterportalapi/public/marker.svg */
+
+const makeMarker = ({ stroke = '#FFF', strokeWidth = '2', fill = '#005CA9' }) =>
```

Comment:
Is there a plan to also make `stroke` and `strokeWidth` configurable? Otherwise, I don't see the necessity to have those two values as parameters. If so, Hhving them as variables would be overblown as well.

Reply:
Yes, we should make them configurable, as well as making the default marker style configurable, too. I've written it down in our internal ticket POLAR-215.

Reply:
Implementing it with the other feature seems reasonable 🎩